mdflynn/overlook
# Feature/dashboards

## What is the change?
- Moves 3 Manager methods and test to the Parent class
- Creates and styles the guest and manager dashboards
- Adds manager search for guest by name ability
- Displays search results with guest's relevant data
- Adds data to the dashboards based on who logged in

## What does it fix?
No fix this PR.

## Is this a fix or a feature?
This is a feature.

## Where should the reviewer start?
- `Manager.js` lines 8-80
- `base.scss` lines 217-342
- `index.js` entire file
## How should this be tested?
- Run `npm start` in your terminal and navigate to http://localhost:8080/ 
- login with either 'manager' or 'customer1' as a username and a password of 'overlook2020' to see different dashboards and functionality.
